src/__snapshots__/schema.spec.ts.snap

```diff
@@ -2,4 +2,193 @@
 
 exports[`schema generate generates valid schema for profile without scope 1`] = `[]`;
 
+exports[`schema generate generates valid schema for profile without scope 2`] = `
```

Comment:
question: why are there two snapshots of the same test?

Reply:
to expects each generates one entry